prow/config/endpointslice.yaml

```diff
@@ -17,5 +17,6 @@ kubeadmConfigPatches:
   controllerManager:
     extraArgs:
       "controllers": "*,endpointslice"
+      "feature-gates": "EndpointSlice=true"
```

Comment:
Is this for e2e test? I think this can be in a separate pr with any new test

Reply:
it's not used in any e2e test (yet).

Reply:
What's the meaning enable it in k8s deploy?

Reply:
yes this is the config to turn on endpoint slice. The change is needed because it 1.17 they made some changes. It's not used for tests but I am going to add a test soon